src/core/server/http/index.ts

```diff
@@ -97,3 +97,9 @@ export type {
 } from './types';
 export { BasePath } from './base_path_service';
 export type { IBasePath } from './base_path_service';
+
+export { cspConfig, CspConfig } from './csp';
+export type { ICspConfig } from './csp';
+
+export { externalUrlConfig, ExternalUrlConfig } from './external_url';
+export type { IExternalUrlConfig, IExternalUrlPolicy } from './external_url';
```

Comment:
Mind the potential conflicts with `IExternalUrlPolicy` as it's been moved in https://github.com/elastic/kibana/pull/135562

cc @gsoldevila